DataFormats/MuonReco/src/Muon.cc

```diff
@@ -133,6 +133,26 @@ int Muon::numberOfMatchedStations( ArbitrationType type ) const
    return stations;
 }
 
+unsigned int Muon::expectedNnumberOfMatchedStations( float minDistanceFromEdge ) const 
+{
+  unsigned int stationMask = 0;
+  for( std::vector<MuonChamberMatch>::const_iterator chamberMatch = muMatches_.begin();
+       chamberMatch != muMatches_.end(); chamberMatch++ )
+    {
+      if (chamberMatch->detector()!=MuonSubdetId::DT && chamberMatch->detector()!=MuonSubdetId::CSC) continue;
+      float edgeX = chamberMatch->edgeX;
+      float edgeY = chamberMatch->edgeY;
+      // check we if the trajectory is well within the acceptance
+      if(edgeX<0 && fabs(edgeX)>fabs(minDistanceFromEdge) &&
```

Comment:
Since edgeX/edgeY have to be negative (i.e. inside) this could be achieved more efficiently if you define
```
minDistanceFromEdge = -fabs(minDistanceFromEdge);
```
out of the loop, and then 
```
 if( edgeX<minDistanceFromEdge && edgeY<minDistanceFromEdge)
```
here

Reply:
This is how it's implemented in other places. I don't see any efficiency problem here. This code is touched once in 5-10 years and having the same implementation will save time for whoever will want to change anything.  So I don't think it's necessary to change it. Am I missing something?

Reply:
I agree that you don't have performance issues for this code, and because of that I won't insist.
On the other hand, when something can straightforwardly get improved, I don't see why not to do it: I'll keep signalling it in my comments when I notice.

Reply:
Thanks for the clarification and for the suggestions - they are always appreciated, just in this case I think it's justified to keep it as-is.